code/game/machinery/requests_console.dm

```diff
@@ -120,9 +120,20 @@ var/list/obj/machinery/requests_console/requests_consoles = list()
 			if(!("[department]" in req_console_information))
 				req_console_information += department
 
+/obj/machinery/requests_console/attack_ghost(user as mob)
+	if(..(user))
```

Comment:
I know you're just copying what other procs in the file do, but it would be better to replace this with `if(..())` so as to not implicitly remove arguments if any are added in a future PR.

Unlike trailing returns, this actually has a chance to break something later on.

Reply:
Removed argument